Mathlib/LinearAlgebra/BilinearMap.lean

```diff
@@ -132,7 +132,8 @@ def flip (f : M →ₛₗ[ρ₁₂] N →ₛₗ[σ₁₂] P) : N →ₛₗ[σ₁
   mk₂'ₛₗ σ₁₂ ρ₁₂ (fun n m => f m n) (fun n₁ n₂ m => (f m).map_add _ _)
     (fun c n  m  => (f m).map_smulₛₗ _ _)
     (fun n m₁ m₂ => by simp only [map_add, add_apply])
-    (fun c n  m  => by simp only [map_smulₛₗ, smul_apply])
+    -- Note: #8386 changed `map_smulₛₗ` into `map_smulₛₗ _`.
```

Comment:
Do we know why these are not unfolding now?

Reply:
Hmm. It doesn’t look like unfolding. It looks like we use up all the assignable metavariables.